### PR TITLE
Tran nguyen branch

### PR DESCRIPTION
**Pull request by trannguyen19**

### CODE REVIEW

**Comment by krishc7** [comment]
Code looks good, could maybe use a few more comments

**Comment by ShivDadh1** [comment]
Nice work. Conditions for arrival and departure as well as input validation are well done.

**Comment by trannguyen19** [comment]
> Code looks good, could maybe use a few more comments

It's true. I should've left more comments in the conditions

File: meal.c [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        printf("Enter the time of departure (example: 5:02 am): ");
+        scanf("%d:%d %s",&depart_hour,&depart_min,time);
+        if ((strcmp(time,"am") == 0) || (strcmp(time,"pm") == 0)){
+            if ( depart_hour >= 1 && depart_hour <= 12) {
+                if ( depart_min >= 0 && depart_min <= 59) {  
+                    break;
+                }
+            }
+        }
+
```
Comment (AraboHovsepian) [inline meal.c:63]: Nice job with the input validation and adding comments to your code

**Pull request merged**